docs/reference.rst

```diff
@@ -684,21 +684,25 @@ bootloader will not select it for booting anymore.
 As shown above this is either the case before an installation to make the
 update atomic from the bootloader's perspective, or optionally after the
 installation and a reboot into the new system, when a service detects that the
-system is in an unusable state. This potentially allows falling back to a
+system is in an unusable state. This potentially allows falling back to another
```

Comment:
I'd leave it as 'a', because we probably only have one working system in that situation.